bcgov/mds
# Mds 4186 create nod tab

# Main

- Created UI and layout for NOD tab
- Created Table for viewing current NODs
- Added create NOD button
- Created create NOD form
  - disabled fields which are not yet implemented
- Implemented NOD redux including state management, api connectivity and form management
- Mirrored redux implementation on Core-web to satisfy automated checks

Not done
- Implement the view buttons on the table view to open a detailed modal of the notice of departure

# Other


# How to test

- Open minespace, and select a mine that you have access to.  On the dashboard, to the right of the permits you should see a `Notices of Departure` tab.  
- Click on the `Create Notice Of Departure` button to open the modal with the form for creating a new notice of departure
- Select a permit number and add a title then click submit
- The new notice of departure should end up in the list view

# Notes

<img width="413" alt="image" src="https://user-images.githubusercontent.com/83598933/161152808-ef2dcead-aec6-4224-857e-605898d8e183.png">
